containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx

```diff
@@ -12,6 +12,9 @@ export interface ColumnInfoInput {
   value?: string;
   className?: string;
   hiddenBaseText?: string;
+  // Generics could be used here instead but we don't want to have to provide it
+  // when this property is optional. Using `any` avoids that issue.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   applyToValue?: (value: any) => any;
```

Comment:
```suggestion
  applyToValue?: (value: string) => string;
```
I'm pretty sure this _should_ be true - does it cause issues?

Reply:
It does. Sometimes we do things like [this](https://github.com/CDCgov/dibbs-ecr-viewer/blob/jw/348/containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx#L169) (`data` is a `ReactNode`) 🙈

Reply:
oh `splitStringWith` is the problem. I wonder if we can just move that after the `applyToValue` part. Most of the functions that are passed to `applyToValue` would choke on those `p` tags, so I don't think this is currently functioning if both of those cases hit

Reply:
There are some additional issues beyond this example as well, such as [this one](https://github.com/CDCgov/dibbs-ecr-viewer/blob/jw/348/containers/ecr-viewer/src/app/view-data/components/EcrDocument/clinical-data.tsx#L363). `formatDate` optionally accepts a string and returns `string | undefined`.

It looks like `toSentenceCase` used in this same file works similarly to `formateDate`.

Reply:
so would `(value: string | undefined) => string | undefined)` cover the usages?

Reply:
No 😭  I keep finding new usages each time I make a change lol. [This one](https://github.com/CDCgov/dibbs-ecr-viewer/blob/jw/348/containers/ecr-viewer/src/app/services/labsService.tsx#L397) returns `RenderableNode`.